.github/workflows/pr-ci.yml

```diff
@@ -73,13 +82,25 @@ jobs:
         registry: ghcr.io/${{ github.repository_owner }}
         username: ${{ github.event.pull_request.user.login }}
         password: ${{ secrets.GHCR_PR_TOKEN }}
-      if: github.repository_owner == 'cryostatio'
+    - name: store image amd64
+      id: amd64_image
+      outputs:
+        amd64_image: ${{ steps.push-to-ghcr.outputs.registry-path }}
+      if: matrix.arch == 'amd64'
+    - name: store image arm64
+      id: arm64_image
+      outputs:
+        amd64_image: ${{ steps.push-to-ghcr.outputs.registry-path }}
+      if: matrix.arch == 'arm64'
     - name: Comment test image link
       uses: thollander/actions-comment-pull-request@v1
       with:
         message: |-
-          Test image available:
+          arm64 test image available:
```

Comment:
Can we use markdown table?

Reply:
| ARCH  | IMAGE                                                                                      |
|-------|--------------------------------------------------------------------------------------------|
| AMD64 | `ghcr.io/cryostatio/cryostat:pr-1599-cadf449151f9e1b5e15bdd3555c91fc97924864b-linux-amd64` |
| ARM64 | `ghcr.io/cryostatio/cryostat:pr-1599-25b3afaf6b2b63be0835808068a335dee918ce0b-linux-arm64` |